pkgs/by-name/po/podman-tui/package.nix

```
@@ -43,9 +42,9 @@ buildGoModule rec {
     [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
 
   passthru.tests.version = testers.testVersion {
```

Not directly related, but you could replace the tester with
```suggestion
  nativeInstallCheckInputs = [
    versionCheckHook
  ];
  versionCheckProgramArg = "version";
  doInstallCheck = true;
```

---

Unfortunately we need https://github.com/NixOS/nixpkgs/pull/411609 to get `HOME` trick working.

---

Maybe the HOME variable is not needed for this program to output version anymore

---

That's what I think as well.